docs/index.html

```diff
@@ -321,7 +303,13 @@ <h1>Generated attributes</h1>
 			</tr>
 		</thead>
 		<tbody>
-			<tr>
+			<tr property="generated-attribute" mv-multiple>
+				<td><code property="name"></code></td>
+				<td property="on" class="markdown"></td>
+				<td property="purpose" class="markdown"></td>
+			</tr>
+
+			<!-- <tr>
```

Comment:
No need to comment things out in case we need them in the future. That's what version control is for! :)